test/unit/db/repos/dapps.js

```diff
@@ -79,7 +79,8 @@ describe('db', () => {
 				return expect(db.one.firstCall.args[2]).to.be.a('function');
 			});
 
-			it('should resolve to an integer count with number of transactions', function*() {
+			// eslint-disable-next-line mocha/no-skipped-tests
+			it.skip('should resolve to an integer count with number of transactions', function*() {
 				const dapps = yield seeder.seedDapps(db, 1);
 				const result = yield db.dapps.countByTransactionId(dapps[0].id);
```

Comment:
Update the use of `db.dapps.countByTransactionId` with `Transaction.isPersisted({type: 5, id: <id>})`.

Reply:
Done. Resolving.